### ISSUE TITLE
context menu: decorative character should be hidden from ATs

### ISSUE THREAD

**Issue by Malvoz** [Maps4HTML/Web-Map-Custom-Element#240]
The decorative character ">" should be hidden from assistive technology.

When focusing the menu item, the [NVDA](https://www.nvaccess.org/) screen reader announces <q>Copy coordinates C greater menu button collapsed submenu</q>.

<img width="550" src="https://user-images.githubusercontent.com/26493779/104386550-b7c99080-5535-11eb-8a60-ec38c066a2f0.png" alt="Decorative character not hidden from screen readers">
